CSharpMath.CoreTests/_Helpers/FrontEnd/TestGlyphNameProvider.cs

```diff
@@ -1,8 +1,33 @@
 namespace CSharpMath.CoreTests.FrontEnd {
-  class TestGlyphNameProvider : Display.FrontEnd.IGlyphNameProvider<char> {
+  using System.Linq;
+  using TGlyph = System.Text.Rune;
+  class TestGlyphNameProvider : Display.FrontEnd.IGlyphNameProvider<TGlyph> {
```

Comment:
Do we know what this is?

Reply:
Looks up a name in latinmodern-math.json

Reply:
Again, copied in your comment. To be in line with the description, should TestGlyphNameProvider be renamed to LatinModernGlyphNameProvider?

Reply:
This is strictly for tests to approximate glyph lookup. The correct and comprehensive implementation of this is in CSharpMath.Apple.